tempus_fugit_minecraft/player.py

```diff
@@ -291,3 +291,25 @@ def keep_player_within_coordinates(dimension, boundary_size=WORLD_SIZE):
             return -boundary_size
         else:
             return dimension
+
+    def slow_walk(self) -> None:
```

Comment:
Change name of function to slow_player_walk_speed

Reply:
Sure, I will change it to `slow_walking_speed` since it's already in the `Player` class.